SolarRender.h

```diff
@@ -0,0 +1,354 @@
+#pragma once
+
```

Comment:
This appears to be duplicate file and should be removed before merging to master.

Reply:
I removed it.